stock_barcodes/wizard/stock_barcodes_read_todo.py

```diff
@@ -41,7 +41,7 @@ class WizStockBarcodesReadTodo(models.TransientModel):
     qty_done = fields.Float(
         "Done",
         digits="Product Unit of Measure",
-        readonly=False,
+        readonly=True,
```

Comment:
@pedrobaeza Maybe indeed here a readonly=True is justified ?

Reply:
But I don't see how changing the field to readonly makes that the wizard now shows the quantity field updated. That should happen the same being readonly or not.

Reply:
@pedrobaeza 
I thought because it's a computed field needed to be readonly = true
I made the change and it worked.

Reply:
No, computed writable fields are readonly=False, and that's something intended for allowing people to change the value. I think you are connecting 2 unrelated things (or at least, not being the proper patch).

Reply:
Yes, it's possible. But I've tried right now and the behavior is like I've described. 
In the wizard quantity done is not update in the wizard view.
If I change the field with 'readonly = True', quantity done is update in the wizard view.

Maybe there's a better solution.

In v15 neither update 'cuantity_done' in the wizard view.